tools/maven/installer.xml

```diff
@@ -30,6 +30,7 @@
             <useDefaultExcludes>true</useDefaultExcludes>
             <excludes>
                 <exclude>%regex[(?!((?!${project.build.directory}/)[^/]+/)*src/)(.*/)?\.gitignore]</exclude>
+                <exclude>/compose/quickstart/zip_folder/*.zip</exclude>
```

Comment:
`${project.basedir}/compose...`

Reply:
Hi, @bossenti! Thanks for the suggestion! I have updated the path in my latest commit :)